src/Microsoft.ML.Data/DataLoadSave/EstimatorChain.cs

```diff
@@ -93,7 +93,10 @@ public EstimatorChain<TNewTrans> Append<TNewTrans>(IEstimator<TNewTrans> estimat
 
         /// <summary>
         /// Append a 'caching checkpoint' to the estimator chain. This will ensure that the downstream estimators will be trained against
-        /// cached data. It is helpful to have a caching checkpoint before trainers that take multiple data passes.
+        /// cached data. It is helpful to have a caching checkpoint before trainers or feature engineering that take multiple data passes.
+        /// It is also helpful to have after a slow operation, for example after dataset loading from a slow source or after feature
```

Comment:
>It is also helpful [](start = 12, length = 18)

Can you also mention that adding a cache checkpoint at the end of the estimator chain is meaningless?